src/main/java10/org/codehaus/plexus/util/VersionSpecifics.java

```diff
@@ -0,0 +1,29 @@
+package org.codehaus.plexus.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.io.Writer;
+
+/**
+ * Implementation specific to Java SE 10 version.
+ */
+final class VersionSpecifics
+{
+    private VersionSpecifics() {
+        // singleton
```

Comment:
I'd say util, not singleton

Reply:
>I'd say util

I actually don't understand what you want to tell me with "I'd say util".